### ISSUE TITLE
journal titles don't display properly in italics

### ISSUE THREAD

**Issue by upossekel** [srophe/e-gedsh#74]
Journal appear again, but not in italics as they are supposed to. Winona, can you please fix that?
Thanks!

**Comment by upossekel** [comment]
Also book titles. Hopefully that is very easily fixed. I'd need to have this work correctly before going on to further proofing.

**Comment by wsalesky** [comment]
Please provide a link and a specific example for me to test. Link to the
entry, and then text as it currently is and the text as it should appear.
Thanks!

**Comment by upossekel** [comment]
Link:http://wwwb.library.vanderbilt.edu/exist/apps/e-gedsh/entry.html?id=http://gedsh.bethmardutho.org/Abdisho-of-Gazarta&num=6

Line 2871, title is 'Literatur' and should be in italics. Bibliography entry for Baumstark.
In same entry, just below, the sic/corr function is still problematic (line 2872). The original read J. Lampart but he is A. Lampart. I corrected according to how Dan instructed. Now both letters appear.
Similarly, in title of Lampart's book I added colon before the word 'Joseph' which does not seem to carry through to the website either.
Thanks, Winona.
